build/vsts_build.yaml

```diff
@@ -542,18 +542,19 @@ phases:
         Remove-Item -Path $(BuildOutputTargetPath) -Force -Recurse -ErrorAction SilentlyContinue
     condition: "eq(variables['Agent.JobStatus'], 'Failed')"
 
-- phase: Functional_Tests_On_Windows
+- job: Functional_Tests_On_Windows
   dependsOn: Initialize_Build
+  timeoutInMinutes: 150
```

Comment:
Should we lower this? 

I think we put it at 150 before we split them up. 

Maybe we should go back to 90?